### PR TITLE
Add small column block (project and grant page)

### LINKED ISSUE #52: Add small column block (project and grant page)

### PR DESCRIPTION
**Pull request by RitwikSrivastava**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #52

Test URLs:
- Original: https://www.sunstar.com/research/project
- Before: https://main--sunstar--hlxsites.hlx.page/_drafts/ritsriva/project
- After: https://coloumnsmall--sunstar--hlxsites.hlx.page/_drafts/ritsriva/project 

- [ ]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://coloumnsmall--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/coloumnsmall&index=0

### CODE REVIEW

**Comment by jindaliiita** [comment]
@RitwikSrivastava What is the need of adding a separate small-column block. Why we are not adding one variant of existing column block ?

**Comment by RitwikSrivastava** [comment]
> @RitwikSrivastava What is the need of adding a separate small-column block. Why we are not adding one variant of existing column block ?

The UI on the project and grant page seems different from the existing coloumn block and hence i thought of creating a new block. 
But yeah, I can definitely try to add this as an variant of the existing column block. Thanks.

File: blocks/columns-small/columns-small.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        width: 100%;
+    }
+
+}
+
+@media (max-width: 61rem) {
+    .columns-small h2,
+    .columns-small:not(.no-buttons) h3,
+    .columns-small:not(.no-buttons) h4 {
+        color: #3C4650;
```
Comment (dnbute) [inline blocks/columns-small/columns-small.css:112]: I think this color is present in `styles.css` as `--light-black`, could you use the variable instead?

**Comment by sdmcraft** [comment]
> The UI on the project and grant page seems different from the existing coloumn block and hence i thought of creating a new block. But yeah, I can definitely try to add this as an variant of the existing column block. Thanks.

I think this may resemble more to a `cards` block. The desktop and mobile both have cards stacked vertically. The desktop has image and text aligned horizontally while the mobile has it aligned vertically.

**Pull request closed without merging**